metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -139,6 +140,8 @@ def create(
     workflow_timeout=None,
     workflow_priority=None,
 ):
+    validate_tags(tags)
```

Comment:
@valayDave You would need to make a similar addition to the Airflow PR once this PR is merged.